### DISCUSSION TITLE
Manager explicitly enabled in "enabledManagers" config, but found no results.

### DISCUSSION THREAD

**Discussion by sudoexec** [renovatebot/renovate#29663] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

I want to use custom managers like [this](https://docs.renovatebot.com/modules/manager/regex/#advanced-capture).
But it looks like renovate don't find custom package files, logs show:
```
DEBUG: Found kubernetes package files (repository=owner/repo)
DEBUG: Found 16 package file(s) (repository=owner/repo)
DEBUG: Manager explicitly enabled in "enabledManagers" config, but found no results. Possible config error? (repository=owner/repo)
       "manager": "custom.regex"
```

I use `npx --yes --package renovate -- renovate-config-validator` and config is ok.Having no idea what's wrong.

Here is my `renovate.json`:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended"
  ],
  "enabledManagers": [
    "custom.regex",
    "kubernetes"
  ],
  "customManagers": [
    {
      "customType": "regex",
      "fileMatch": [
        "^\\.renovate/renovate\\.txt$"
      ],
      "matchStrings": [
        "depName=(?<depName>.+?)packageName=(?<packageName>.+?)datasource=(?<datasource>[a-z-]+?)versioning=(?<versioning>[a-z-]+?)version=(?<currentValue>.+?)"
      ]
    }
  ],
  "kubernetes": {
    "fileMatch": [
      "\\.yaml$"
    ]
  }
}
```

Here is `.renovate/renovate.txt` in the repo root path:
```txt
depName=pocketbase packageName=pocketbase/pocketbase datasource=github-releases versioning=semver-coerced version=v0.22.12
```

**Comment by rarkins** [comment, marked as answer]
You don't need to use enabledManagers. It's only useful if you want to limit managers to a select few.

If you do use it, and any of the enabled managers don't return dependencies, then you'll get this warning. Most likely it's because your matching isn't working

**Reply by sudoexec** [reply]
Thanks for your help.
You're right. It's a silly regex error.
